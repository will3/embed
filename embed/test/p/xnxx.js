const test = require('../utils/test');

test('xnxx', 
	'http://www.xnxx.com/video-d4ltdd7/0/finally_got_to_fuck_haydee_from_high_school', 
	'http://static-hw.xvideos.com/swf/xv-player.swf?id_video=22049905');