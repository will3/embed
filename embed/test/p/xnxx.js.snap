// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` xnxx 1`] = `
Object {
  "originalUrl": "http://www.xnxx.com/video-d4ltdd7/0/finally_got_to_fuck_haydee_from_high_school",
  "site_name": undefined,
  "thumbnail": Object {
    "url": "http://img-hw.xvideos.com/videos/thumbs169xnxxll/d4/18/0c/d4180c1564788a6195df0c72df73458e/d4180c1564788a6195df0c72df73458e.19.jpg",
  },
  "title": "Finally got to fuck Haydee from High School - XNXX.COM",
  "video": Object {
    "description": undefined,
    "height": "400",
    "title": undefined,
    "url": "http://static-hw.xvideos.com/swf/xv-player.swf?id_video=22049905",
    "width": "510",
  },
}
`;
